service/upload/upload.go

```diff
@@ -1,12 +1,15 @@
 package uploadService
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/NyaaPantsu/nyaa/config"
 	"github.com/NyaaPantsu/nyaa/model"
 )
 
+const trackerRegex = `^(http[s]*|udp)://[a-z0-9\.:\-/?]+$`
```

Comment:
Can't you have an attempt on inserting other data in the db by falsifying a .torrent file?
For torrent tracker url, I just tried the regex with the ones in our config.
If you think we don't need to be sure that it is a tracker url, then I can just remove the regex :)